context/app/static/js/components/entity-search/Search/Search.jsx

```diff
@@ -41,7 +43,14 @@ function Search() {
   const variables = useSearchkitVariables();
   const { results } = useSearchkitSDK(config, variables, transporter, filters);
 
-  return results?.hits ? <ResultsTable hits={results.hits} /> : null;
+  return (
+    <SearchLayout>
+      <SidebarLayout>
+        {results?.facets && results.facets.map((facet) => <SelectFacet key={facet.identifier} facet={facet} />)}
+      </SidebarLayout>
+      <ResultsLayout>{results?.hits && <ResultsTable hits={results.hits} />}</ResultsLayout>
```

Comment:
Down the road would these both be ternaries? ... but shouldn't there always be facets, even if no results are returned?

Reply:
So long as there isn't an error the results object will always be returned. Hits are the actual 'results' which can be empty.